lib/Transforms/Scalar/UnifyAssumes.cc

```diff
@@ -105,9 +139,21 @@ bool UnifyAssumesPass::runOnFunction(Function &F) {
   assumeFlag = B.CreateAlloca(B.getInt1Ty(), 0U, nullptr, "assume.flag");
   B.CreateStore(B.getTrue(), assumeFlag);
 
+  // -- process all orphan asserts first
```

Comment:
Don't understand the concept of `orphan` assert, so marking anything related to it as to discuss

Reply:
removed. uneeded.